src/Microsoft.TestPlatform.Build/Microsoft.TestPlatform.targets

```diff
@@ -31,7 +31,7 @@ Copyright (c) .NET Foundation. All rights reserved.
 
     <Microsoft.TestPlatform.Build.Tasks.VSTestTask
```

Comment:
Is there a doc on how dotnet test works internally? I assume that this being named a vstest target has nothing to do with `dotnet vstest` and that flows through the VSTestTask invoke. Hence your change would solve both dotnet test and dotnet vstest?

Reply:
dotnet vstest would not be affected. This is only for dotnet test flows where project file or sln file or a folder is specified. 
dotnet vstest verb is nothing but dotnet vstest.console and usage allows only dlls and not proj or sln files.